swagger/v2/swagger.yaml

```diff
@@ -994,8 +994,10 @@ paths:
 
                 $InformationPreference = 'Continue'
                 $ErrorActionPreference = 'Stop'
+                # https://github.com/PowerShell/PowerShell/issues/14348
+                $ProgressPreference = 'SilentlyContinue'
 
-                Write-Information "Acoustic Workbench downloader script version 4.9.0"
+                Write-Information "Acoustic Workbench downloader script version 4.9.1"
```

Comment:
Are you manually updating these version numbers?

Reply:
Nah, it's happening naturally with our build.

I did run `generate_docs.sh` manually to see the changes to the swagger file in this commit - that is usually left for our release stage but I wanted to see the changes were working.